model/status.go

```diff
@@ -22,7 +22,7 @@ type Status struct {
 	Status         string `json:"status"`
 	Manual         bool   `json:"manual"`
 	LastActivityAt int64  `json:"last_activity_at"`
-	ActiveChannel  string `json:"active_channel" db:"-"`
```

Comment:
This looks like it'll remove the `active_channel` field from the existing API. What are the consequences of this from a backwards compatibility point of view?